src/Endpoint.js

```diff
@@ -220,63 +230,76 @@ const parseSchemaName = (ref) => {
+          <strong><code>{name}</code></strong>
+        </Heading>
+      </Box>
+      <Box flex={true} pad={{ horizontal: 'small' }} margin={{ vertical: 'small' }}>
+        <Markdown>
+          {sanitizeForMarkdown(response.description)}
+        </Markdown>
       </Box>
-      {refs.schema && parseSchemaName(refs.schema.$ref) &&
-        <Box direction='column' align='start' pad={{ bottom: 'medium' }}>
-          <Text>
-            {'returns '}
-            <strong>
-              <RoutedAnchor
-                label={parseSchemaName(refs.schema.$ref)}
-                path={`/definition?name=${parseSchemaName(refs.schema.$ref)}`}
-              />
-            </strong>
-          </Text>
-        </Box>
-      }
-      {response.examples ?
-        Object.keys(response.examples).map(key =>
-          <Schema key={key} label={key} data={data} schema={response.examples[key]} />)
-        :
-        <Schema data={data} schema={getExample(response)} />
-      }
     </Box>
-  );
+    {refs.schema && parseSchemaName(refs.schema.$ref) &&
+      <Box direction='column' align='start' pad={{ bottom: 'medium' }}>
+        <Text>
+          {'returns '}
+          <strong>
+            <RoutedAnchor
+              label={parseSchemaName(refs.schema.$ref)}
+              path={`/definition?name=${parseSchemaName(refs.schema.$ref)}`}
+            />
+          </strong>
+        </Text>
+      </Box>
+    }
+    {response.examples ?
+      Object.keys(response.examples).map(key =>
+        <Schema key={key} label={key} data={data} schema={response.examples[key]} />)
+      :
+      <Schema data={data} schema={getExample(response)} />
+    }
+  </Box>
+);
 
 const Header = ({
   data, executable, methodName, subPath,
 }) => (
-    <Box direction='row' align='center' wrap={true}>
-      <Box background='brand' pad={{ horizontal: 'medium', vertical: 'xsmall' }}>
-        <Text size='xlarge'>
-          <strong>{methodName.toUpperCase()}</strong>
-        </Text>
-      </Box>
-      <Box margin={{ horizontal: 'small' }}>
-        <Text size='xlarge' color='brand'>
-          {data.basePath || (data.servers && data.servers.length > 0 && data.servers[0].url)}{subPath}
-        </Text>
-      </Box>
-      {executable ? <LinkNext color='brand' /> : null}
+  <Box direction='row' align='center' wrap={true}>
+    <Box background='brand' pad={{ horizontal: 'medium', vertical: 'xsmall' }}>
+      <Text size='xlarge'>
+        <strong>{methodName.toUpperCase()}</strong>
+      </Text>
+    </Box>
+    <Box margin={{ horizontal: 'small' }}>
+      <Text size='xlarge' color='brand'>
+        {data.basePath || (data.servers && data.servers.length > 0 && data.servers[0].url)}{subPath}
+      </Text>
     </Box>
-  );
+    {executable ? <LinkNext color='brand' /> : null}
+  </Box>
+);
 
 class Method extends Component {
+  componentDidMount() {
+    const baseURI = window.location.href.split('#');
```

Comment:
`window` doesn't exist outside of a browser. I'd recommend running a check with `if (typeof window !== 'undefined)` before this.